src/rust/engine/logging/src/logger.rs

```diff
@@ -112,13 +146,26 @@ impl PantsLogger {
 
 impl Log for PantsLogger {
   fn enabled(&self, metadata: &Metadata) -> bool {
-    metadata.level() <= max_level()
+    let global_level: LevelFilter = { *self.global_level.lock().borrow() };
```

Comment:
Is calling `*foo.borrow()` the same thing as `*&foo`, which I would expect to be the same thing as `foo` but I think isn't?

Reply:
`borrow` is a method defined on `RefCell` (https://doc.rust-lang.org/std/cell/struct.RefCell.html#method.borrow). However, I don't think I actually need `Mutex<RefCell<T>>`, just a `Mutex` should be fine here, so I'm removing this call.